server/services/utils/alertFlagLabels.ts

```diff
@@ -0,0 +1,118 @@
+import { Alert, Prisoner } from '../../data/interfaces/prisoner'
+import { AlertFlagLabel } from '../interfaces/establishmentRollService/AlertTypeLabel'
+
```

Comment:
With more prisoner specific stuff in this repo, we're starting to need to do some copy-paste of code from the profile.  So if we need to add an alert flag, it'll now need updating in two places.  One to keep an eye out for I suppose!

Reply:
Yeah that was also the case when this was in dps. I think we can look towards packaging stuff up when we get a bit of time and sharing common functionality. May be useful precedent to set in general for DPS